plothist/plotters.py

```diff
@@ -532,20 +544,15 @@ def plot_comparison(
             )
         ax.axhline(1, ls="--", lw=1.0, color="black")
         ax.set_ylabel(r"$\frac{" + x1_label + "}{" + x2_label + "}$")
-        if comparison_ylim is None:
-            ax.set_ylim(0.0, 2.0)
-        else:
-            ax.set_ylim(comparison_ylim)
 
     elif comparison == "pull":
         ax.axhline(0, ls="--", lw=1.0, color="black")
         ax.set_ylabel(
             rf"$\frac{{ {x1_label} - {x2_label} }}{{ \sqrt{{\sigma^2_{{{x1_label}}} + \sigma^2_{{{x2_label}}}}} }} $"
         )
         if comparison_ylim is None:
-            ax.set_ylim(-5.0, 5.0)
-        else:
-            ax.set_ylim(comparison_ylim)
+            comparison_ylim = (-5.0, 5.0)
+        ax.set_ylim(comparison_ylim)
```

Comment:
This should be done only once at the end of the function, together with `set_xlim()`